Content.Shared/Atmos/GasMixture.cs

```diff
@@ -271,6 +271,24 @@ public bool Equals(GasMixture? other)
                    && Volume.Equals(other.Volume);
         }
 
+        /// <summary>
+        /// Returns True if supersetMix has more moles for all gases by mole quantity only.
+        /// </summary>
+        /// <param name="supersetMix"></param> 
+        /// <param name="subsetMix"></param>
```

Comment:
What is this?

Reply:
my IDE being a jerk :P Ill fix it real fast.